### PR TITLE
Samaritan 2142/lcv beacon renewal complete

### PR DESCRIPTION
**Pull request by dannysperry**
[Trello Card](https://trello.com/c/trWsONEr/2142-lcv-beacon-renewal-complete)

**Description**
- Added the "Thank you" screen that shows after a user clicks "Skip Form & Renew"
- Changes the 2 column individual view to initially be full width and the LCV component slides open over the profile from the right.

### CODE REVIEW

File: src/components/individuals/index.js [hunk 1]
```diff
@@ -205,6 +230,11 @@ class Individuals extends Component {
         let homelessTimelineLoader = this.renderHomelessTimeline()
         // let graphData = this.renderGraph()
 
+        let rightFrameClasses = [classes.rightFrame]
+        if (this.state.drawerOpen) {
+            rightFrameClasses.push(classes.rightFrameOpen)
```
Comment (cvkumar) [inline src/components/individuals/index.js]: Nit: Would it be better to add logic like this in the component's render itself? Vs. using just the CSS property. Just a thought.

File: src/containers/lifecare_visit.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import {connect} from 'react-redux'
+import * as actions from '../redux/actions/index'
+import LifecareVisit from '../components/lifecare_visit'
+
+// state is object that holds reducers from combineReducers and hence the state in them
+function mapStateToProps(state) {
```
Comment (cvkumar) [inline src/containers/lifecare_visit.js]: Nice work on this

File: src/components/lifecare_visit/index.js [hunk 3]
```diff
@@ -45,22 +48,59 @@ class LifecareVisit extends Component {
       }
     }
 
+    reset() {
+      let middle = this.ref.current.clientHeight * .67
+      this.setState({
+        locked: false
+      })
+      this.ref.current.scroll({top: middle, left: 0, behavior: 'smooth'})
+      this.ref.current.addEventListener('scroll', this.handleScroll)
```
Comment (cvkumar) [inline src/components/lifecare_visit/index.js]: I believe you can use react built-in events here instead.

File: src/components/individuals/index.js (hunk 1 above)
Comment (dannysperry) [reply src/components/individuals/index.js]: I think doing it that way makes animation harder.

**Pull request merged**